docs/guides/integration-mobile-client-side.md

```diff
@@ -49,10 +49,10 @@ To integrate with UID2 client-side, you'll need to complete the following steps:
 
 ## Mobile SDK Version
 
-This guide provides instructions for using version 1.2.0 or higher of either of these UID2 mobile SDKs:
+This guide provides instructions for using either of these UID2 mobile SDKs:
 
-- SDK for Android
-- SDK for iOS
+- SDK for Android(version 1.5.0 or later)
+- SDK for iOS(version 1.7.0 or later)
```

Comment:
Same here, just add a space:
- SDK for Android (version 1.5.0 or later)
- SDK for iOS (version 1.7.0 or later)